src/server/server.rs

```diff
@@ -302,6 +302,17 @@ impl<I, E> Builder<I, E> {
         self
     }
 
+    /// Set a timeout for reading client request headers. If a client does not 
+    /// transmit the entire header withing this time, the connection is closed.
+    ///
+    /// Default is None.
+    #[cfg(feature = "http1")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "http1")))]
+    pub fn http1_header_read_timeout(mut self, read_timeout: Duration) -> Self {
```

Comment:
Should we make the argument be an `impl Into<Option<Duration>>`, so people can explicitly turn it off if they want? Is that ever a good idea?

Reply:
I'm not sure. The default value is already `None`, so staying with the style of the other methods never setting it should be the way of disabling it?